src/routes/structures/[slug]/quick-start.svelte

```diff
@@ -8,22 +8,26 @@
   import {
     canShowQuickStart,
     isStructureInformationsComplete,
-    isFirstSearchDone,
     hasOneService,
-    hasAtLeastTwoMembers,
-    saveQuickStartDone,
+    hasAtLeastTwoMembersOrInvitedMembers,
   } from "./quick-start";
-  import type { Structure, StructureMember } from "$lib/types";
-  import { userInfo } from "$lib/utils/auth";
+  import type {
+    Structure,
+    StructureMember,
+    PutativeStructureMember,
+  } from "$lib/types";
+  import { modifyStructure } from "$lib/requests/structures";
 
   export let structure: Structure;
   export let members: StructureMember[];
+  export let putativeMembers: PutativeStructureMember[];
 
   let showQuickStart = canShowQuickStart(structure);
 
-  function hideQuickStart() {
-    saveQuickStartDone(structure.slug);
-    showQuickStart = false;
+  async function hideQuickStart() {
+    await modifyStructure({ ...structure, quickStartDone: true });
```

Comment:
modifyStructure fait un PATCH, donc tu n'as pas besoin en théorie de renvoyer toutes ses données
```suggestion
    await modifyStructure({ quickStartDone: true });
```